### ISSUE TITLE
Error handling for monitored queries

### ISSUE THREAD

**Issue by YashMeh** [scorelab/senz#97]
**Issue**
There is no proper mechanism for error handling of the queries which can be monitored through the admin panel.

**Solution**
- Create an endpoint in the senz-web/backend which returns different status codes for different queries.
- These status codes can then be mapper from a errorMapper Map in the scala project .
- Customised error messages can then be sent to the user.

**Expected Mapper**

```bash
ErrorMapper = {
  500: "OK",
  501: "#ERR->USER_NOT_AUTHORISED",
  502: "#ERR->PROJECT_NOT_VALID",
  503: "#ERR->USER_PROJECT_NOT_COMPATIBLE",
  504: "#ERR->SENDER_RECEIVER_NOT_COMPATIBLE",
  505: "#ERR->DEVICE_OFFLINE",
  506: "#ERR->PROJECT_OFFLINE",
  507: "#ERR->SENDER_RECEIVER_SAME"
};
```
**Mock**
While sending a monitored query user should get error messages according to the signature ,sender and receiver given in the query.

**Comment by YashMeh** [comment]
The issue requires a different approach that what given above .

**Comment by YashMeh** [comment]
**Issue**
Creating a standalone query checker for the senz-switch
**Solution**
Create a map of public keys and lookup through it to maintain integrity with the senz-web.

**Comment by YashMeh** [comment]
closed via #115
